src/test/regress/sql/subselect.sql

```diff
@@ -517,3 +517,50 @@ fetch backward all in c1;
 
 commit;
 --end_ignore
+
+-- Ensure that ORCA correctly builts Query dxl tree and all attributes inside
+-- the nested SubLink, which are referenced in GROUP BY clause, are processed
+-- during query normalization. The inner part of SubPLan should contain only t.j
+-- start_ignore
+drop table if exists t;
+drop table if exists s;
+-- end_ignore
+create table t (i int, j int) distributed by (i);
+create table s (i int, j int) distributed by (i);
```

Comment:
I suggest to put dropping and creating of the s table to the first usage of this table. First two tests do not use s. Dropping at the end should be in reverse order

Reply:
done